code/game/objects/items/grenades/plastic.dm

```diff
@@ -84,6 +84,8 @@
 	if(bomb_target != user && HAS_TRAIT(user, TRAIT_PACIFISM) && isliving(bomb_target))
 		to_chat(user, span_warning("You don't want to harm other living beings!"))
 		return
+	if(istype(bomb_target, /obj/item/gun/grenadelauncher))
```

Comment:
It would make more sense for the c4 to be blacklisted from being inserted into the grenade launcher.  It barely behaves like a grenade.

Reply:
I thought loading C4 into grenade launcher is its intended feature, but I will close this if it is not and try doing what you suggest.

Reply:
Done, I changed it. Just need to edit the page now.